src/components/button/index.tsx

```diff
@@ -49,11 +51,16 @@ const Button: FC<Props> = ({ children, ...props }) => {
     isDisabled = false,
     as = 'button',
     isExternal,
+    leftIcon,
+    rightIcon,
     ...rest
   } = props;
 
   return (
     <ChakraButton
+      leftIcon={leftIcon}
+      rightIcon={rightIcon}
```

Comment:
I double checked with Enrique and we could have the case for a right icon on the button.